tests/test_service.py

```diff
@@ -134,6 +134,16 @@ def test_list_wishlists(self):
         self.assertEqual(data[2]['customer_id'], 101)
         self.assertEqual(data[2]['id'], 3)
         self.assertEqual(data[2]['name'], "wishlist_name3")
+        
+    def test_get_wishlist(self):
```

Comment:
can you also add tests for when wishlist is not found?

Reply:
done